playbooks/code42_alert_response_playbook.json

```diff
@@ -0,0 +1,7088 @@
+                            }
+                        },
+                        "position": {
+                            "x": -520,
+                            "y": 60
+                        },
+                        "previous_function": "",
+                        "previous_name": "prompt_further_investigation",
+                        "respond_in": "30",
+                        "response_key": "Message",
+                        "response_options": [],
+                        "response_type": "list",
+                        "responses": [
+                            {
+                                "response_key": "Yes/No",
+                                "response_options": [
+                                    "Yes",
+                                    "No"
+                                ],
+                                "response_prompt": "Further investigate?",
+                                "response_type": "list"
+                            },
+                            {
+                                "response_key": "Message",
+                                "response_options": [],
+                                "response_prompt": "Case name",
+                                "response_type": "message"
+                            }
+                        ],
+                        "show_number": true,
+                        "size": {
+                            "height": 80,
+                            "width": 80
+                        },
+                        "state": "prompt",
+                        "status": "",
+                        "type": "coa.Prompt",
+                        "warn": false,
+                        "z": 266
+                    },
+                    {
+                        "action": "create case",
+                        "action_type": "investigate",
+                        "active": false,
+                        "active_keys": {},
+                        "active_values": {
+                            "assignee": "",
+                            "case_name": "prompt_further_investigation:action_result.summary.responses.1",
+                            "description": "",
+                            "findings": "",
+                            "subject": "get_alert_details_1:action_result.data.*.actor"
+                        },
+                        "angle": 0,
+                        "app": "Code42",
+                        "appid": "4d8f53a7-7b12-4d7d-8b01-6575680acf6f",
+                        "approver": "",
+                        "assets": [
+                            {
+                                "action": "create case",
+                                "actions": [
+                                    "run advanced query",
+                                    "run query",
+                                    "hunt file",
+                                    "add case event",
+                                    "list cases",
```

Comment:
For some reason, the new action `get user profile` does not appear in this list.
I see it action list assets though.